contribs/emissions/src/main/java/org/matsim/contrib/emissions/WarmEmissionAnalysisModule.java

```diff
@@ -105,38 +105,36 @@ public WarmEmissionAnalysisModule(
 					// If this causes too many problems, we could insert a switch (or attach it to the fallback behavior switch).  kai, feb'20
 					// Eventually vehicle category and vehicle attribute should be alligned in order to make the allCombinations setting useful
 					// see discussion in  https://github.com/matsim-org/matsim-libs/issues/1226 kturner, nov'20
-					if (detailedHbefaWarmTable != null) {
```

Comment:
is it ensured before, that it is not null? If yes, it is ok for removing. If not I would let it there (Or do you have changed here anything else in this block, what I do not see in the github diff view?)

Reply:
This is tested before already and was therefore superfluous.